environments/conda-linux-64.lock.yml

```diff
@@ -94,34 +94,34 @@ dependencies:
   - coverage=7.6.9=py312h178313f_0
   - crashtest=0.4.1=pyhd8ed1ab_1
   - croniter=3.0.4=pyhd8ed1ab_0
-  - cryptography=43.0.3=py312hda17c39_0
-  - curl=8.10.1=hbbe4b11_0
+  - cryptography=44.0.0=py312hda17c39_0
```

Comment:
Last week this package was downgraded which [resolved the RTD build failure](https://github.com/catalyst-cooperative/pudl/pull/3989#discussion_r1878385203). This PR updates the package back to `44.0.0` and the RTD build is passing.